code/game/machinery/vending/vendor_types/requisitions.dm

```diff
@@ -208,13 +208,33 @@
 			updateUsrDialog()
 			return //We found our item, no reason to go on.
 
+//Special cargo-specific vendor with vending offsets
+/obj/structure/machinery/cm_vending/sorted/cargo_guns/cargo
+	vend_flags = VEND_CLUTTER_PROTECTION | VEND_LIMITED_INVENTORY //We want to vend to turf not hand, since we are in requisitions
+
+/obj/structure/machinery/cm_vending/sorted/cargo_guns/cargo/get_appropriate_vend_turf(var/mob/living/carbon/human/H)
+	var/turf/T
```

Comment:
Better var name

Reply:
Renamed all 3 instances of `turf/T` in `get_appropriate_vend_turf` functions.